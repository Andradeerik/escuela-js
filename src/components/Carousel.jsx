import React from 'react';
import '../styles/components/Carousel.scss';

const Carousel = ({ children }) => (
  <div className="carousel">
    <div className="carousel__container">{children}</div>
  </div>
);

export default Carousel;
