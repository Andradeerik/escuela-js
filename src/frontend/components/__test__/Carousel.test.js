import React from 'react';
import { create } from 'react-test-renderer';
import Carousel from '../Carousel';

describe('Carousel component', () => {
  test('Matches the snapshot', () => {
    const carousel = create(<Carousel />);
    expect(carousel.toJSON()).toMatchSnapshot();
  });
});
