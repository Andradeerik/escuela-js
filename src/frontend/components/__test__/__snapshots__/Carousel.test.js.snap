// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel component Matches the snapshot 1`] = `
<div
  className="carousel"
>
  <div
    className="carousel__container"
  />
</div>
`;
